### ISSUE TITLE
NVDA doesn't read descriptions until key is released

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/friction#296]
**Test device**
Dell

**Operating System**
Win10

**Browser**
Firefox

**Problem description**
For https://github.com/phetsims/qa/issues/791, when a WASD key or arrow key is held down to move the book, NVDA doesn't read out any descriptions. No descriptions are read out until I release the key, but this results in many descriptions being missed. 
This does not match was happens with VO or what I see in ally view. 

**Steps to reproduce**
1. With NVDA on, move the book all the way to one side keeping the key held down. 
2. Release the key

**Visuals**

https://user-images.githubusercontent.com/87318828/162016451-c27c8cd1-9eac-4a15-bbe5-c8b52a840158.mov

**Comment by stemilymill** [comment]
The behavior is the same on Win10 chrome + JAWS

**Comment by zepumph** [comment]
This is actually the behavior that we expect to see. I'm not sure how your results differ in VO, but in general you only get description upon the end of an interaction, not during it. It is most typical for a user to move more judiciously to receive info as they go. Can you please describe the behavior of VO, if it it seems buggy given this information?

**Comment by Nancy-Salpepi** [comment]
@zepumph I am not hearing any responsive descriptions at all when I move the book except for "right/left". Something else must be wrong at the moment, but I don't see anything in the console. 

https://user-images.githubusercontent.com/87318828/171997739-7ad27181-81ed-4814-b542-0566281cad4e.mov

**Comment by zepumph** [comment]
I believe that https://github.com/phetsims/utterance-queue/commit/b18b2586d0ca2bbc1db01d5bc53b40a2ae2a50b3 is the culprit.

**Comment by zepumph** [comment]
That commit was NOT the cuplrit. @jessegreenberg and I investifated and bugs reported and to be fixed in https://github.com/phetsims/utterance-queue/issues/82. 

@Nancy-Salpepi, marking this on hold until you can test description again.

**Comment by zepumph** [comment]
Ok, we should have description back in Friction. @Nancy-Salpepi can you please take this issue from https://github.com/phetsims/friction/issues/296#issuecomment-1144112364.

**Comment by Nancy-Salpepi** [comment]
@zepumph I'm still not getting any description with Friction on master (I cleared my history and am in an a private window).

**Comment by Nancy-Salpepi** [comment]
@zepumph I start to hear alerts with VO while still holding down a key. This is how VO has always behaved for me (I think). I see the same thing in John Travoltage. 

https://user-images.githubusercontent.com/87318828/173071095-32039d47-0b24-41fe-94ac-98d94c28c124.mov

**Comment by zepumph** [comment]
As far as I know this is just a difference between screen readers, and not something to investigate. I also poked around the internet and didn't find anything. I see that the response is making it to the screen reader (NVDA menu -> tools -> speech viewer), but then keeping the key down immediately silences it. @jessegreenberg can you confirm that this different isn't a bug? I'm happy to investigate further if you disagree.

**Comment by zepumph** [comment]
@terracoda and I discussed this today, we agree this isn't a bug. @Nancy-Salpepi please note this discrepancy between screen readers. Nothing to change here. Closing
